dbt/adapters/databricks/relation_configs/tblproperties.py

```diff
@@ -28,6 +28,7 @@ class TblPropertiesConfig(DatabricksComponentConfig):
         "pipeline_internal.catalogType",
         "pipelines.metastore.tableName",
         "pipeline_internal.enzymeMode",
+        "clusteringColumns",
```

Comment:
cluster by columns are stored in tblproperties.  We don't want users attempting to manipulate them outside of specifying liquid_clustered_by